templates/resources/provisioner.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+Title: "Provisioner resources in a management cluster"
+Description: |-
+  Fetching provisioner resources in a management cluster.
+---
+
+# Management Cluster Provisioner
+
+Read provisioner in a management cluster using this Terraform module.
```

Comment:
Is it just read operation?

Reply:
No, it is read for data source and not resource. Updated the same